ddtrace/contrib/algoliasearch/patch.py

```diff
@@ -101,7 +100,7 @@ def _patched_search(func, instance, wrapt_args, wrapt_kwargs):
     if not pin or not pin.enabled():
         return func(*wrapt_args, **wrapt_kwargs)
 
-    with pin.tracer.trace('algoliasearch.search', service=pin.service, span_type=SEARCH_SPAN_TYPE) as span:
+    with pin.tracer.trace('algoliasearch.search', service=pin.service) as span:
```

Comment:
In cases like `algoliasearch` where we don't as yet use a more general span type, we should not set one, and leave the default (`None`) as is.